server/service/osquery_utils/queries.go

```diff
@@ -130,32 +130,14 @@ var hostDetailQueries = map[string]DetailQuery{
 				return nil
 			}
 
-			if strings.Contains(strings.ToLower(rows[0]["name"]), "ubuntu") {
-				// Ubuntu takes a different approach to updating patch IDs so we instead use
-				// the version string provided after removing the code name
-				regx := regexp.MustCompile(`\(.*\)`)
-				vers := regx.ReplaceAllString(rows[0]["version"], "")
-				host.OSVersion = fmt.Sprintf(
-					"%s %s",
-					rows[0]["name"],
-					strings.TrimSpace(vers),
-				)
-			} else if rows[0]["major"] != "0" || rows[0]["minor"] != "0" || rows[0]["patch"] != "0" {
-				host.OSVersion = fmt.Sprintf(
-					"%s %s.%s.%s",
-					rows[0]["name"],
-					rows[0]["major"],
-					rows[0]["minor"],
-					rows[0]["patch"],
-				)
-			} else {
-				host.OSVersion = fmt.Sprintf(
-					"%s %s",
-					rows[0]["name"],
-					rows[0]["build"],
-				)
-			}
-			host.OSVersion = strings.Trim(host.OSVersion, ".")
```

Comment:
Have we double checked that this doesn't impact negatively OVAL? I think we had an issue like it before, where we changed how we parsed/stored a version and then we ended up skipping OVAL checks.

Reply:
Juan recently expanded our test coverage for how the OVAL package uses the os version and this PR is just extracting the parser to a separate function so I think we should be good here.

@juan-fdz-hawa would you mind giving this a quick triple check?